DEVELOPMENT.md

```diff
@@ -3,14 +3,27 @@
 * [Java 21](https://openjdk.org/install)
 * [Docker](https://docs.docker.com/engine/install)
 * [Node.js 18](https://nodejs.org/en/download) (or higher)
+* [Google Chrome](https://www.google.com/chrome/)
 * [Intellij Idea](https://www.jetbrains.com/help/idea/installation-guide.html) (optional)
 
-## Clone this repo
+### On Linux
+  * Docker BuildX
+  * Docker composer
+
+## Working with repositories
+
+If you want to send PR, create a fork of the repo.
+
+Then clone it from your account.
 
 ```shell
 git clone --depth 1 git@github.com:tolgee/tolgee-platform.git
 ```
 
+When working, create a dev branch and work on that.
+
+Before sending PR, squash all your commits in to one, and then push to your repo first, then make the PR
```

Comment:
Why do you think this is necessary?

Reply:
If we create a separate branch, then we can squash the intermediate commits to a single one in the main branch. That way, we are not crowding the upstream with those intermediate commits. But we can keep track of our own progress in the internal branch.